libs/agno/agno/agent/agent.py

```diff
@@ -4812,7 +4812,16 @@ def get_system_message(
 
         # 3.2.5 Add information about agentic filters if enabled
         if self.knowledge is not None and self.enable_agentic_knowledge_filters:
-            valid_filters = getattr(self.knowledge, "valid_metadata_filters", None)
+            valid_filters, invalid_keys = self.knowledge.validate_filters(self.knowledge_filters)
```

Comment:
But agentic filters override the provided filters? That is kinda the point? So it should get _all_ valid filters